public/app/features/provisioning/Shared/TokenPermissionsInfo.tsx

```diff
@@ -14,55 +14,33 @@ export function TokenPermissionsInfo() {
           Go to{' '}
           <TextLink external href="https://github.com/settings/personal-access-tokens/new">
             GitHub Personal Access Tokens
-          </TextLink>
-          . Make sure to include these permissions under <b>Repository</b>:
+          </TextLink>{' '}
+          and click <strong>"Fine-grained token"</strong>. Make sure to include these permissions:
         </Trans>
       </div>
 
-      <table className={styles.permissionTable}>
-        <tbody>
-          <tr className={styles.headerSeparator}>
-            <th>
-              <Trans i18nKey="provisioning.token-permissions-info.permission">Permission</Trans>
-            </th>
-            <th>
-              <Trans i18nKey="provisioning.token-permissions-info.access">Access</Trans>
-            </th>
-          </tr>
-          <tr>
-            <td>
-              <Trans i18nKey="provisioning.token-permissions-info.contents">Contents</Trans>
-            </td>
-            <td>
-              <Trans i18nKey="provisioning.token-permissions-info.read-and-write">Read and write</Trans>
-            </td>
-          </tr>
-          <tr>
-            <td>
-              <Trans i18nKey="provisioning.token-permissions-info.metadata">Metadata</Trans>
-            </td>
-            <td>
-              <Trans i18nKey="provisioning.token-permissions-info.readonly">Read-only</Trans>
-            </td>
-          </tr>
-          <tr>
-            <td>
-              <Trans i18nKey="provisioning.token-permissions-info.pull-requests">Pull requests</Trans>
-            </td>
-            <td>
-              <Trans i18nKey="provisioning.token-permissions-info.read-and-write">Read and write</Trans>
-            </td>
-          </tr>
-          <tr>
-            <td>
-              <Trans i18nKey="provisioning.token-permissions-info.webhooks">Webhooks</Trans>
-            </td>
-            <td>
-              <Trans i18nKey="provisioning.token-permissions-info.read-and-write">Read and write</Trans>
-            </td>
-          </tr>
-        </tbody>
-      </table>
+      <ul className={styles.permissionsList}>
+        <li>
+          <Trans i18nKey="provisioning.token-permissions-info.content-permission">
+            Content: <span className={styles.accessLevel}>Read and write</span>
```

Comment:
do the new strings show up in the translation files somehow?

Reply:
Yeah, altho now this makes me think - GitHub doesn't support other languages, so the permission names will be all in English and shouldn't be translated 🤔